controllers/noobaa-rules.yaml

```
@@ -1,56 +1,58 @@
+# https://prometheus.io/docs/practices/rules/#naming-and-aggregation
+# https://github.com/noobaa/noobaa-operator/blob/08a6a4e23e04ef635c6d060298a90ee57b3b67df/deploy/crds/noobaa.io_noobaas_crd.yaml#L14
  groups:
    - name: noobaa-telemeter.rules
      rules:
        - expr: |
-           NooBaa_num_buckets_claims{namespace="redhat-data-federation"}
+           max(NooBaa_num_buckets_claims{namespace="redhat-data-federation"}) without (instance)
```

(nit) I personally prefer the `max without(...) ()` version over `max () without(...)` because it makes it more obvious which labels are dropped/retained. But it's only an indication, feel free to keep it like that if you want.

```suggestion
           max without(instance) (NooBaa_num_buckets_claims{namespace="redhat-data-federation"})
```

The other suggestion would be explicitly list the label(s) you want to retain (e.g. namespace) vs. dropping only the instance label. Again not a deal breaker for me.

```suggestion
           max by(namespace) (NooBaa_num_buckets_claims{namespace="redhat-data-federation"})
```